core/src/main/java/org/apache/iceberg/actions/BaseCopyTableActionResult.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.actions;
+
+public class BaseCopyTableActionResult implements CopyTable.Result {
```

Comment:
this should probably be similar to how all the other Result classes are implemented, such as https://github.com/apache/iceberg/blob/82e0a56323537432e087a07b4cfd5100bf37073d/core/src/main/java/org/apache/iceberg/actions/BaseDeleteReachableFiles.java#L23-L33